### PR TITLE
New Plugins!

### PR DESCRIPTION
**Pull request by Airbus-A330**
Added some new plugins that are extremely necessary for Sencord!

* `jumpscare` - Keeps you on your toes!
* `wigglyText` - Adds wiggly text to your client!
* `systemNotifications` - Ensures you hear all your official system notifications from Discord!

Merge this pull request now!

### CODE REVIEW

**Review by sinjs — CHANGES_REQUESTED** [review]
Fix yo tests 🙏🙏

File: src/plugins/systemNotifications/index.tsx
Comment (sinjs) [inline src/plugins/systemNotifications/index.tsx:1]: The fuck is this supposed to be

Comment (Airbus-A330) [reply src/plugins/systemNotifications/index.tsx:1]: Sometimes, you might miss an important system message from Discord! This plugin ensures you can not only see but hear a Discord Official System message announcement! It is imperative to Discord's functionality and your user experience that you immediately read all system messages coming to you.

**Review by sinjs — APPROVED** [review]

File: src/plugins/systemNotifications/index.tsx
Comment (sinjs) [reply src/plugins/systemNotifications/index.tsx:1]: Can u write a description for it that makes sense pls

**Comment by sinjs** [comment]
Looks Gay To Me! 🏳️‍🌈

File: src/plugins/systemNotifications/index.tsx
Comment (Airbus-A330) [reply src/plugins/systemNotifications/index.tsx:1]: This plugin plays a loud noise every time a Discord system message comes in. Said noise contains 5 loud horn-like pulses with an audible message explaining a new Official System Message from Discord has been sent to you while simultaneously encouraging you to read the system message immediately. This audio pattern is played twice.

File: src/plugins/systemNotifications/index.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { FluxDispatcher, UserStore } from "@webpack/common";
+
+export default definePlugin({
+    name: "System Notifications",
+    description: "Ensure you are always up to date with the latest system notifications from Discord.",
+    authors: [Devs.Airbus],
+    required: true,
```
Comment (sinjs) [inline src/plugins/systemNotifications/index.tsx:15]: NAHHHHH

File: src/plugins/systemNotifications/index.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    start() {
+       FluxDispatcher.subscribe("USER_UPDATE", ({ user }) => {
+            if (user.id === UserStore.getCurrentUser().id) {
+                this.notify();
```
Comment (sinjs) [inline src/plugins/systemNotifications/index.tsx:20]: Bro forgot to check if it’s a system message

**Review by sinjs — CHANGES_REQUESTED** [review]

File: src/plugins/systemNotifications/index.tsx (hunk 2 above)
Comment (Airbus-A330) [reply src/plugins/systemNotifications/index.tsx:20]: image

Comment (Airbus-A330) [reply src/plugins/systemNotifications/index.tsx:20]: #LGTM #ReadyToMerge #FirstPR #InternPR #Crowdstrike #FirstCrowdstrikePR #SystemFailure !!!

Comment (sinjs) [reply src/plugins/systemNotifications/index.tsx:20]: Bro WHAT

**Comment by Airbus-A330** [comment]
Please don't fire me! I added the check!

**Review by sinjs — APPROVED** [review]

**Comment by sinjs** [comment]
Go back to Vietnam

File: src/plugins/systemNotifications/index.tsx (hunk 2 above)
Comment (Airbus-A330) [reply src/plugins/systemNotifications/index.tsx:20]: Hope this doesn't crash millions of Windows systems! 😅 🤞

**Comment by sinjs** [comment]
Layoff.

**Comment by Airbus-A330** [comment]
Another hard day's work at Sencord 👏 🧹 😄 ❗

**Comment by sinjs** [comment]
As your CEO, you're fired for racism.

**Pull request merged**